### PR TITLE
Bavet122

### PR DESCRIPTION
**Pull request by MassimoVilla**

### CODE REVIEW

File: VaticanBAV/et/BAVet122.xml [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+                           </note>
+                        </msItem>                     
+                     </msItem>      
+                     <colophon xml:lang="am" xml:id="coloph1">
+                        <locus target="#149v"/>
+                        ተፈጸመ፡ ሥርዓተ፤ ቅዳሴ፤ በዘመነ፤ ማርቆስ፡ በወርኃ፤ ኅዳር፤ <date calendar="ethiopian" when-custom="1905">፲፱፻፭ አመተ፤ ምሕረት፨</date> 
+                        ጸሕፊሁ፡ <persName role="scribe" ref="PRS14615Maqoyi"><roleName type="title">አለቃ፡</roleName> መቆዪ</persName>፨                       
+                        <note>
+                           The colophon states that the manuscript was written in the month of <foreign xml:lang="gez">Ḫədār</foreign> of the year of mercy 
+                           <date calendar="ethiopian" when-custom="1905">1905 EC</date>, i.e.
```
Comment (thea-m) [inline VaticanBAV/et/BAVet122.xml:245]: `add @when="1913" to <date>`

File: VaticanBAV/et/BAVet122.xml [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+                     </colophon>
+                  </msItem>
+               </msContents>
+          
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <watermark>
+                           </watermark>
```
Comment (thea-m) [inline VaticanBAV/et/BAVet122.xml:258]: could you delete watermark? I know it's copied from elsewhere, But I think its presence might be confusing in a parchment codex

File: VaticanBAV/et/BAVet122.xml [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+                           <material key="leather"/>                           
+                        </decoNote>
+                        <decoNote xml:id="b3" type="Other">Blind tooled ornament on the leather cover. The ornaments have the shape of a cross and palms.</decoNote>
+                     </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate when="1913">
+                        According to <ref target="#colophon1"/>, the manuscript was written in the year of mercy
```
Comment (thea-m) [inline VaticanBAV/et/BAVet122.xml:385]: coloph1

File: VaticanBAV/et/BAVet122.xml [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+                  </handDesc>
+                     
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <decoNote xml:id="b1">Two wooden boards covered with reddish leather.</decoNote>
+                        <decoNote xml:id="b2" type="bindingMaterial">
+                           <material key="wood"/>   
+                           <material key="leather"/>                           
+                        </decoNote>
+                        <decoNote xml:id="b3" type="Other">Blind tooled ornament on the leather cover. The ornaments have the shape of a cross and palms.</decoNote>
```
Comment (thea-m) [inline VaticanBAV/et/BAVet122.xml:378]: blind-tooled

File: VaticanBAV/et/BAVet122.xml [hunk 5]
```diff
@@ -0,0 +1,453 @@
...
+                        <locus from="21r" to="50v"/>
+                        <title ref="LIT1960Mashaf#SerataQeddase">Ordinary of the Mass</title>                        
+                        <incipit xml:lang="gez">
+                           <locus target="#21r"/>
+                           በስመ፡ አብ፡ <gap reason="omitted" resp="PRS4805Grebaut PRS9530Tisseran"/> ንጽሕፍ፡ ሥርዓተ፡ ቅዳሴ፡ ከመ፡ ይበሉ፡ ቀሲስ፡ ወዲያቆን፡ ወሕዝብ፡ ወኵሉ፡ 
+                           ዘመፍትው፡ በበጊዜሁ፡ በከመ፡ ሥርዓተ፡ አበዊነ፡ ቅዱሳን፡
+                        </incipit>
+                        <note>The Ordinary of the Mass is separated from the anaphoras, like in recent manuscripts.
+                           The text bears a second title after the prayers for the blessing of the tools for the Eucharist:
+                           <q xml:lang="gez">
```
Comment (thea-m) [inline VaticanBAV/et/BAVet122.xml:137]: @eu-genia I am not sure if q can be used outside of additions or decorations (I believed that not, but could not find it in the guidelines) or if foreign should be used, could you refresh my memory?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, wonderful, some minor comments and one questions that's more for Zhenia :)

File: VaticanBAV/et/BAVet122.xml (hunk 5 above)
Comment (eu-genia) [reply VaticanBAV/et/BAVet122.xml:137]: I believe `q` is fine anywhere

**Review by eu-genia — APPROVED** [review]
thank you, can be merged after @thea-m 's corrections are in

**Review by thea-m — APPROVED** [review]

**Pull request merged**